app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ActiveRecord::Base
+  has_many :games
+end
```

Comment:
You know what, I am still a little confused by the associations.  I've now added has_many :pieces in both game and user for good measure in the new commit.  Thanks!